### DISCUSSION TITLE
Desktop - show what list a starred project is in

### DISCUSSION THREAD

**Discussion by marcrleonard** [community/community#39369] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I found that on the iOS app it will show what list a particular starred project is in, but, it does not do this on desktop. I really liked seeing this feature in mobile, as its nice to know if you haven't added a project to list - it would be great to have it on desktop.

... Sidenote: I LOVE this feature. I've always been overwhelmed that the starred listing is just a big flat dumping ground.

**Comment by Preetiraj3697** [comment]
Thankyou for feedback @marcrleonard . I try it. 😊
